app/controllers/api/v1/orders_controller.rb

```diff
@@ -65,15 +65,7 @@ def sort_by_urgency(records)
       api :GET, '/v1/designations/1', "Get a order"
       def show
         root = is_browse_app? ? "order" : "designation"
-        render json: @order,
-          serializer: serializer,
-          root: root,
-          exclude_code_details: true,
-          include_packages: bool_param(:include_packages, true),
-          include_order: false,
-          include_territory: true,
-          include_images: true,
-          exclude_stockit_set_item: true
+        render json: Api::V1::OrderSerializer.new(order_with_orders_count, root: root).to_json
```

Comment:
can you change `to_json` to `as_json` - `as_json` returns a hash and then render will convert it correctly

Reply:
I am trying it out.